roles/deploy/tasks/main.yml

```diff
@@ -2,6 +2,10 @@
 - name: Initialize
   deploy_helper: path="{{ project_root }}" state=present
 
+- name: Update git remote URL
+  shell: "cd {{ project_source_path }} && git remote set-url origin {{ project_git_repo }} || :"
+  register: git_status
+
```

Comment:
As it is right now, the task will always be marked as changed, and will alway update the remote url, even if it hasn't changed. Something like this would prevent that and would only update the url if the project has already been created and the remote changed:

```
- name: Check if project folder exists
  stat: path={{ project_source_path }}
  register: git_project_path

- name: Get remote URL
  cmd: git config --get remote.origin.url 
  args:
    chdir: {{ project_source_path }}
  register: remote_origin_url
  when: git_project_path.stat.exists

- name: Update git remote URL
  cmd: git remote set-url origin {{ project_git_repo }}
  args:
    chdir: {{ project_source_path }}
  when: remote_origin_url is defined and remote_origin_url != project_git_repo
```

I just wrote that on top of my head, so there may be syntax errors that slipped by.